### DISCUSSION TITLE
conversation_assignment liquid attachment url error

### DISCUSSION THREAD

**Discussion by alfredo-callpicker** [chatwoot/chatwoot#4965] [General]
When somebody assign to agent in any Chat with attachments an email is sent.
Into the email If you try to click some attachment to view  the url can not open. [using file_url]

**Comment by sojan-official** [comment]
can you share the following information? 

- is it a self-hosted installation?
- the current version of chatwoot you are using
-  the installation environment 
-  do you have a storage option configured / which?

**Reply by alfredo-callpicker** [reply]
1.- Yes.
2.- The current newer version, i think is 2.+.
3.- Amazon Linux 2 AMI
4.- ACTIVE_STORAGE_SERVICE local
